extensions/amp-story/1.0/amp-story.css

```diff
@@ -921,6 +922,16 @@ amp-story .amp-video-eq,
   --i-amphtml-interactive-theme-border: rgba(0, 0, 0, 0) !important;
 }
 
+.i-amphtml-story-interactive-component[chip-style="transparent"] {
+  --i-amphtml-interactive-options-chip-background-color: transparent !important;
+  --interactive-prompt-background: transparent !important;
+  --interactive-prompt-alignment: center;
```

Comment:
Does this mean adding chip-style=transparent change the default alignment of the prompt, but we let publishers override it?

Reply:
Yes! Ran it through Hong and it looks better to center it by default if there's no prompt background, but they can override it. For binary polls we also center it by default for instance, because it also looks better.

Reply:
Got it, it becomes a bit unpredictable but if we can be easily overridden by publishers config I assume it's fine.